pymtl3/stdlib/rtl/queues.py

```diff
@@ -89,17 +91,17 @@ def up_rdy_signals():
         s.enq_rdy = s.count < s.num_entries
```

Comment:
how about ```s.enq_rdy = (s.count < s.num_entries) & ~s.reset```

Reply:
alright

Reply:
should we prioritize performance or readability in our stdlib?

Reply:
I guess it is not that unreadable compared to the if s.reset case.